tools/checkcomments/checkcomments.go

```diff
@@ -17,19 +17,21 @@ package main
 
 import (
 	"context"
+	"errors"
 	"flag"
 	"log"
 	"regexp"
-	"strconv"
 	"strings"
 
 	"github.com/FerretDB/gh"
 	"golang.org/x/tools/go/analysis"
 	"golang.org/x/tools/go/analysis/singlechecker"
+
+	"github.com/FerretDB/FerretDB/tools/github"
```

Comment:
The tools module should not import the FerretDB module

Reply:
Are you talking about this specific line? It imports the `tools/github module`

Reply:
Right, importing the github package from the tools module is fine